packages/engine/src/scene/systems/ShadowSystem.tsx

```diff
@@ -429,9 +432,8 @@ const updateDropShadowTransforms = () => {
     const sizeBias = 0.3
     const finalRadius = sizeBias * dropShadow.radius + dropShadow.radius * centerCorrectedDist * 0.5
 
-    const shadowMaterial = (getComponent(dropShadow.entity, GroupComponent)[0] as any).material as Material
-    shadowMaterial.opacity = Math.min(1 / (1 + centerCorrectedDist), 1) * 0.6
-
+    const shadowMaterial = (getComponent(dropShadow.entity, GroupComponent)[0] as Mesh).material as Material
+    shadowMaterial.opacity = Math.min(1 / (1 + centerCorrectedDist), 1) * 2
```

Comment:
i wonder why the size here needs to change, my guess is something with our bounding boxes or similar is affecting this. this value was tweaked to look good previously.

Reply:
yes, i the old opacity made the drop shadows almost invisible, this gives a darker shadow